.github/workflows/nodejs.yml

```
@@ -14,22 +14,22 @@ env:
 
 jobs:
   lint:
-    name: Lint on Node.js ${{ matrix.node }} and ${{ matrix.os }}
+    name: Lint on Node.js ${{ matrix.node }}
 
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
 
     strategy:
       matrix:
         node: [12]
-        os: [ubuntu-latest]
 
     steps:
       - uses: actions/checkout@v2
 
       - name: Use Node.js ${{ matrix.node }}
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node }}
```

```suggestion
          node-version: "lts/*"
```

Let's lint on LTS, and save us from having to bump this number in the future.